docs/content/preview/troubleshoot/nodes/check-logs.md

```diff
@@ -56,3 +56,22 @@ For YB-Master and YB-TServer, the log rotation size is controlled by the `--max_
 For YSQL, you also have the additional `postgres*log` files. These logs have daily and size-based log rotation, that is a new log file will be created each day or a log reaches 10 MB size.
 
 For available configuration flags, see [YB-Master logging flags](../../../reference/configuration/yb-master/#logging-flags) and [YB-TServer logging flags](../../../reference/configuration/yb-tserver/#logging-flags).
+
+## Log format
+
+YB-Master and YB-TServer log messages have this form:
+
+```
+Lmmdd hh:mm:ss.uuuuuu threadid file:line] msg
```

Comment:
Missing a left square bracket here. Which parts are optional?

Reply:
There is no left square bracket, that's the actual log format. There are no optional parts.